app/src/main/java/org/blitzortung/android/data/DataHandler.kt

```diff
@@ -63,7 +60,13 @@ class DataHandler @JvmOverloads constructor(private val wakeLock: PowerManager.W
         }
     }
 
-    private val internalDataConsumer: (DataEvent) -> Unit = { dataEvent -> dataConsumerContainer.storeAndBroadcast(dataEvent) }
+    private val internalDataConsumer: (DataEvent) -> Unit = {
+        dataEvent ->
+            if(dataEvent is StatusEvent)
+                dataConsumerContainer.broadcast(dataEvent)
+            else
+                dataConsumerContainer.storeAndBroadcast(dataEvent)
+    }
```

Comment:
Maybe we should only store ResultEvents. ClearDataEvent is obsolete and all status updates can be ignored as well.